dla_cnn/desi/uniform_snr

```diff
@@ -0,0 +1,44 @@
+def split_bins(data_array,num):
```

Comment:
Put this and the other methods like it into `training_sets.py`

Reply:
Then remove this file.

Reply:
Got it